File: chatwoot/java/src/main/java/oseg/chatwoot_examples/UpdateCannedResponseInAccountExample.java
package oseg.chatwoot_examples;

import com.chatwoot.client.ApiException;
import com.chatwoot.client.Configuration;
import com.chatwoot.client.api.*;
import com.chatwoot.client.auth.*;
import com.chatwoot.client.JSON;
import com.chatwoot.client.model.*;

import java.io.File;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class UpdateCannedResponseInAccountExample
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();
        config.setApiKey("USER_API_KEY");

        var cannedResponseCreateUpdatePayload = new CannedResponseCreateUpdatePayload();
        cannedResponseCreateUpdatePayload.content(null);
        cannedResponseCreateUpdatePayload.shortCode(null);

        try
        {
            var response = new CannedResponseApi(config).updateCannedResponseInAccount(
                null, // accountId
                null, // id
                cannedResponseCreateUpdatePayload // data
            );

            System.out.println(response);
        } catch (ApiException e) {
            System.err.println("Exception when calling CannedResponseApi#updateCannedResponseInAccount");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: chatwoot/kotlin/src/main/kotlin/oseg/chatwoot_examples/UpdateCannedResponseInAccountExample.kt
package oseg.chatwoot_examples

import com.chatwoot.client.infrastructure.*
import com.chatwoot.client.apis.*
import com.chatwoot.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class UpdateCannedResponseInAccountExample
{
    fun updateCannedResponseInAccount()
    {
        ApiClient.apiKey["userApiKey"] = "USER_API_KEY"

        val cannedResponseCreateUpdatePayload = CannedResponseCreateUpdatePayload(
            content = null,
            shortCode = null,
        )

        try
        {
            val response = CannedResponseApi().updateCannedResponseInAccount(
                accountId = null,
                id = null,
                _data = cannedResponseCreateUpdatePayload,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling CannedResponseApi#updateCannedResponseInAccount")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling CannedResponseApi#updateCannedResponseInAccount")
            e.printStackTrace()
        }
    }
}
